packages/utils/src/__tests__/array.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { countDistinct, min, max } from '../array';
+
+describe('array', () => {
+  const testData = [
+    ['countDistinct', [], 0],
+    ['single item ', [0], 1],
+    ['multiple items, same value', [0], 1],
+    ['multiple items, different values', [0, 1], 2],
+    ['multiple items, mixed same & different values', [0, 1, 0, 1, 1, 2], 3],
+  ];
+  describe('countDistinct', () => {
+    it.each(testData)('%s', (string, input, expected) => {
+      expect(countDistinct(input)).toBe(expected);
+    });
+
+    describe('custom mappers', () => {
+      const customMappersTestData = [
```

Comment:
And then this can become

```suggestion
      const testData = [
```